automation-tests/test/scratchOrg.ts

```diff
@@ -0,0 +1,347 @@
+/*
+ * Copyright (c) 2023, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+import child_process from 'child_process';
+import fs from 'fs';
+import path from 'path';
+import util from 'util';
+import {
+  DefaultTreeItem,
+  InputBox,
+  QuickOpenBox,
+  Workbench
+} from 'wdio-vscode-service';
+import {
+  EnvironmentSettings
+} from './environmentSettings';
+import {
+  utilities
+} from './utilities';
+
+const exec = util.promisify(child_process.exec);
+
+export class ScratchOrg {
+  private testSuiteSuffixName: string = undefined;
```

Comment:
these can't be assigned to undefined unless it's added to the type definition.

Reply:
I added strict to tsconfig, and the members in scratchOrg.ts and environmentSettings.ts have been updated